Sources/StreamChatSwiftUI/ChatChannel/Polls/CreatePollView.swift

```diff
@@ -119,7 +119,7 @@ public struct CreatePollView: View {
                         if viewModel.multipleAnswers {
                             HStack(alignment: .textFieldToggle) {
                                 VStack(alignment: .leading, spacing: 6) {
-                                    Text(L10n.Composer.Polls.typeNumberFrom1And10)
+                                    Text(L10n.Composer.Polls.typeNumberFrom2And10)
```

Comment:
you would need to change it here as well, if you go with min-max range

Reply:
Updated 👍